2023-02-28 17:13:08.822 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration browser_mod which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:13:08.825 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:13:08.828 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration dyson_local which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:13:08.831 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration tautulli which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:13:08.834 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration webrtc which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:13:24.801 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Living Room' for apple_tv integration not ready yet: Not found at 10.0.1.21, waiting for discovery; Retrying in background
2023-02-28 17:13:46.362 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:13:46.372 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:13:52.736 ERROR (MainThread) [homeassistant.components.sensor] The version platform for the sensor integration does not support platform setup. Please remove it from your config.
2023-02-28 17:13:52.757 ERROR (MainThread) [homeassistant.components.sensor] The version platform for the sensor integration does not support platform setup. Please remove it from your config.
2023-02-28 17:14:02.821 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform darksky is taking over 10 seconds.
2023-02-28 17:14:02.870 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform influxdb is taking over 10 seconds.
2023-02-28 17:14:02.896 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform file is taking over 10 seconds.
2023-02-28 17:14:04.789 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform rest is taking over 10 seconds.
2023-02-28 17:14:04.793 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform rest is taking over 10 seconds.
2023-02-28 17:14:04.798 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform rest is taking over 10 seconds.
2023-02-28 17:14:04.801 WARNING (MainThread) [homeassistant.components.binary_sensor] Setup of binary_sensor platform nest is taking over 10 seconds.
2023-02-28 17:14:04.809 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform nest is taking over 10 seconds.
2023-02-28 17:14:06.257 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:06.262 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:08.891 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:08.897 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:09.746 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.81:11311 ssl:default [Connect call failed ('10.0.40.81', 11311)]
2023-02-28 17:14:09.762 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.81:11311 ssl:default [Connect call failed ('10.0.40.81', 11311)]
2023-02-28 17:14:11.376 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.80:9078 ssl:default [Connect call failed ('10.0.40.80', 9078)]
2023-02-28 17:14:11.521 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:12.789 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Notify Battery - Replace Low Battery' trigger: In 'numeric_state' condition: unknown entity sensor.august_web_garage_door_keypad_battery
2023-02-28 17:14:29.154 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:29.164 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:31.179 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.80:9078 ssl:default [Connect call failed ('10.0.40.80', 9078)]
2023-02-28 17:14:31.194 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.80:9078 ssl:default [Connect call failed ('10.0.40.80', 9078)]
2023-02-28 17:14:31.336 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.80:9078 ssl:default [Connect call failed ('10.0.40.80', 9078)]
2023-02-28 17:14:31.776 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.81:11311 ssl:default [Connect call failed ('10.0.40.81', 11311)]
2023-02-28 17:14:31.797 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:32.306 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:32.603 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.80:9078 ssl:default [Connect call failed ('10.0.40.80', 9078)]
2023-02-28 17:14:32.612 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.81:11311 ssl:default [Connect call failed ('10.0.40.81', 11311)]
2023-02-28 17:14:32.882 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: 
2023-02-28 17:14:43.031 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.80:9078 ssl:default [Connect call failed ('10.0.40.80', 9078)]
2023-02-28 17:14:43.259 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.81:11311 ssl:default [Connect call failed ('10.0.40.81', 11311)]
2023-02-28 17:14:43.292 WARNING (MainThread) [homeassistant.components.konnected.panel] Exception trying to connect to panel: Cannot connect to host 10.0.40.81:11311 ssl:default [Connect call failed ('10.0.40.81', 11311)]
2023-02-28 17:16:34.883 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration browser_mod which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:16:34.894 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:16:34.897 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration dyson_local which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:16:34.901 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration tautulli which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:16:34.905 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration webrtc which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-02-28 17:31:49.672 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/zwave_js_server/client.py", line 116, in async_send_command
    return await future
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/zwave_js/update.py", line 164, in _async_update
    await self.driver.controller.async_get_available_firmware_updates(
  File "/usr/local/lib/python3.10/site-packages/zwave_js_server/model/controller/__init__.py", line 719, in async_get_available_firmware_updates
    data = await self.client.async_send_command(
  File "/usr/local/lib/python3.10/site-packages/zwave_js_server/client.py", line 118, in async_send_command
    self._result_futures.pop(message_id)
KeyError: '85350d450b304eaf9790229208bd18cd'
2023-02-28 17:31:49.646 ERROR (Recorder) [root] Uncaught thread exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/src/homeassistant/homeassistant/components/recorder/core.py", line 654, in run
    self._run_event_loop()
  File "/usr/src/homeassistant/homeassistant/components/recorder/core.py", line 670, in _run_event_loop
    self._shutdown()
  File "/usr/src/homeassistant/homeassistant/components/recorder/core.py", line 1208, in _shutdown
    self._end_session()
  File "/usr/src/homeassistant/homeassistant/components/recorder/core.py", line 1202, in _end_session
    self.run_history.clear()
  File "/usr/src/homeassistant/homeassistant/components/recorder/run_history.py", line 140, in clear
    assert self._current_run_info.end is not None
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/attributes.py", line 482, in __get__
    return self.impl.get(state, dict_)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/attributes.py", line 942, in get
    value = self._fire_loader_callables(state, key, passive)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/attributes.py", line 973, in _fire_loader_callables
    return state._load_expired(state, passive)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/state.py", line 712, in _load_expired
    self.manager.expired_attribute_loader(self, toload, passive)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/loading.py", line 1369, in load_scalar_attributes
    raise orm_exc.DetachedInstanceError(
sqlalchemy.orm.exc.DetachedInstanceError: Instance <RecorderRuns at 0x7fb55e4ceb60> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: https://sqlalche.me/e/14/bhk3)
